Tests/Autogenerated/Mock.generated.swift

```diff
@@ -21,8 +21,12 @@ class CBCentralManagerMock: NSObject {
 
     var retrievePeripheralsParams: [([UUID])] = []
     var retrievePeripheralsReturns: [[CBPeripheralMock]] = []
+    var retrievePeripheralsReturn: [CBPeripheralMock]?
     func retrievePeripherals(withIdentifiers identifiers: [UUID]) -> [CBPeripheralMock] {
         retrievePeripheralsParams.append((identifiers))
+        if let retrievePeripheralsReturn = retrievePeripheralsReturn {
```

Comment:
Why is it necessary? If you want default return value, maybe we can use it after all return values from array are used?

Reply:
:+1: